terminus-ui/src/csv-entry/csv-entry.component.ts

```diff
@@ -73,7 +73,6 @@ let nextUniqueId = 0;
  *              maxRows="1000"
  *              columnCount="6"
  *              rowCount="12"
- *              fullWidth=false
```

Comment:
IMO we should not remove this. Up till now, the primary JSDoc for a component has every possible input/output regardless of the default value.

Reply:
Added it back in, as well as other inputs that were not captured.